docs/best_practices/general.rst

```diff
@@ -22,32 +27,38 @@ Group. In this case the instances must have unique names. If they are both equal
 the class name (e.g. "ElectricalSeries_1" and "ElectricalSeries_2"). If one of the instances is an extra of less
 importance, name that one something different (e.g. "ElectricalSeries" and "ElectricalSeries_extra_electrode").
 
-Names are not for storing meta-data. If you need to place other data of the same neurodata_type, you will need to
-choose another name. Keep in mind that meta-data should not be stored solely in the name of objects. It is OK to name
-an object something like “ElectricalSeries_large_array” however the name alone is not sufficient documentation. In this
-case, the source of the signal will be clear from the device of the rows from the linked electrodes table region, and
-you should also include any important distinguishing information in the description field of the object. Make an effort
-to make meta-data as explicit as possible. Good names help users but do not help applications parse your file.
+
+
+.. _best_practice_name_slashes:
+
+No Not Use Slashes
+~~~~~~~~~~~~~~~~~~
 
 ’/’ is not allowed in names. When creating a custom name, using the forward slash (/) is not allowed, as this can
```

Comment:
```suggestion
’/’ is not allowed in Group or Dataset names, as this can
```

Reply:
Do references to Groups and Datasets still make sense on the MatNWB side?

Reply:
Yes, because those are [HDMF schema language ](https://hdmf-schema-language.readthedocs.io/en/latest/) concepts (and by extension, NWB schema language concepts).